ui/src/css/core/flex.sass

```diff
@@ -135,9 +135,9 @@
       > *
         margin-left: $size
     &-y-#{$name}
-      margin-top: -$size
+      margin-top: -$size #{"/* rtl:ignore */"}
```

Comment:
any non horizontal related CSS does not gets translated for RTL purposes, so this addition should in theory not matter

Reply:
I know it should not have been processed, but unfortunately it is.

Reply:
Just checked. It doesn't process it. Those lines are not getting doubled.

Reply:
This is an extract from `quasar.rtl.css`:

```css
[dir] .q-gutter-y-none, [dir] .q-gutter-none {
  margin-top: 0;
}
[dir] .q-gutter-y-none > *, [dir] .q-gutter-none > * {
  margin-top: 0;
}
```

Reply:
The "dir" attribute is assigned to the `<html>` tag by Quasar. So it's fine.

Reply:
I know, but a rule with dir gets higher priority than one without. So a child of .q-gutter-y-md cannot have it's vertical padding changed by a .q-my-lg